themes/default/content/docs/using-pulumi/pulumi-packages/troubleshoot-packages.md

```diff
@@ -0,0 +1,73 @@
+---
+title_tag: "How to Debug Pulumi Provider Packages"
+meta_desc: "Learn the process for debugging Pulumi providers locally."
+title: Debug providers
+h1: Debug Pulumi providers
+meta_image: /images/docs/meta-images/docs-meta.png
+menu:
+    usingpulumi:
+        parent: pulumi-packages
+        identifier: how-to-debug-providers
+        weight: 2
+aliases:
+- /docs/guides/pulumi-packages/how-to-debug-providers/
```

Comment:
Since this is a new doc, we don't really need an alias (we can always use `keywords` to hook additional Algolia search queries). I'd also suggest having the URL be named `debugging-provider-packages.md`, as "troubleshoot" doesn't really apply here (debugging is something that's done during the normal course of development).

```suggestion
```